prisma/schema.prisma

```diff
@@ -10,11 +10,25 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
+model User {
+  id        Int      @id @default(autoincrement())
+  name      String
+  username  String   @unique
+  password  String
+  createdAt DateTime @default(now())
+  Role      Role     @default(MANAGER)
+}
+
+enum Role {
```

Comment:
don't constraint yourself with database level roles. changing this in the long run will be harder, rather opt to string based roles because more dynamic.

Never program defensively.